### DISCUSSION TITLE
[FR]: option to set a default value for a field

### DISCUSSION THREAD

**Discussion by www-wagner** [RafaelGB/obsidian-db-folder#547] [Ideas]
### Present your request

I miss an option to initialize a field with a default value.

For example, if I have a `task` table with a `status` field with the following possible values: `queue`, `doing` and `done`. It'd be wanted that all new rows initialize `status` with default value `queue`.

I've used an example for the `Select` field type but this setting could be used at any field type.

### For which platform do you request this request??

Cross

**Comment by imed-ghomari** [comment]
Couldn't this be solved by using templates?

Edit: or you want a native way to do this ?

**Comment by www-wagner** [comment]
Yes, a native way in the fields setting.

![image](https://user-images.githubusercontent.com/18124316/197242318-754120ff-c9c9-41c3-b505-2b3e17a9b72f.png)

**Comment by RafaelGB** [comment]
I think the correct way is to use the create-a-row templates.  Including more specific rules within each column also conflicts directly with the templates themselves.

**Comment by RafaelGB** [comment]
I will open a discussion for your idea
